psi4/src/psi4/libmints/mcmurchiedavidson.cc

```diff
@@ -160,4 +160,77 @@ void fill_M_matrix(int maxam, int maxpow, const Point& PC, double a, double b, s
     }
 }
 
+void fill_R_matrix(int maxam, double p, const Point& P, const Point& C, std::vector<double>& R,
```

Comment:
Can you document the expected size of `R` somewhere?

Reply:
Done.